redis/src/aio/multiplexed_connection.rs

```diff
@@ -556,6 +646,16 @@ impl MultiplexedConnection {
             _ => Ok(vec![value]),
         }
     }
+
+    /// Gets [`crate::caching::CacheStatistics`] for current connection.
+    #[cfg(feature = "cache-aio")]
+    pub fn get_cache_statistics(&self) -> CacheStatistics {
```

Comment:
nit: this should probably be an `Option<CacheStatistics>`, so that `None` is returned if there's no cache manager. (and adjust docs)